### PR TITLE
Add all page sign in eSign

### PR DESCRIPTION
**Pull request by iAziz786**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -943,20 +966,20 @@ Content-Type: application/json
...
+| webhook_url       | true      | A valid POST API URL, Detailed response will be sent here for success or failures |
+| purpose           | true      | The reason for the transaction                                                    |
+| phone_override    | false     | If you wanted to update your phone number                                         |
+| document_required | true      | The document for which you wanted to get the details                              |
+| { ITR             | false     | Specify for how many years of ITR you wanted to fetch                             |
+| 26AS }            | false     | Specify for how many years of 26AS you wanted to fetch                            |
+| phone             | true      | Phone number you wanted to use at our platform                                    |
+| pan               | true      | PAN number linked to ITR portal                                                   |
+| dob               | true      | Date of birth of the PAN holder in DD-MM-YYYY format                              |
+| pdf_required      | false     | Whether you need PDF of ITR fetched                                               |
```
Comment (iAziz786) [inline README.md:982]: Please ignore this part, it just formatting change. No need to review this part.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ AadhaarAPI | ZOOP web SDK for E-sign and Bank Statement Analysis Gateway
 
 # Table of Contents
 
-## Zoop E-Sign Gateway (v3 Beta).
+## Zoop E-Sign Gateway (v3).
```
Comment (iAziz786) [inline README.md:7]: Removed eSign v3 from Beta. I hope that's ok.

File: README.md [hunk 3]
```diff
@@ -189,6 +190,28 @@ Content-Type: application/json
 | purpose     | Purpose of document signature                                                                   | Mandate as per norms. Will be used to generate consent text and logged in DB. |
 | responseURL | POST API URL where the Agency receives the response after the e-signing is completed.           | A valid POST API URL, else response back to your server will fail.            |
 
+<a name="esignSignAllPages"></a>
+
+#### 4.3.1 Sign on All Pages
+
+You can also attach signature on all the pages of the document. The signature will be attached on the **same coordinate of every page**. In order to attach the signature on every page you need to provide `signPageNumber` as 0 in the init call.
+
+<pre><code class="language-json">{
```
Comment (thefoodiecoder) [inline README.md:199]: @iAziz786 Add co-ordinate part in the example so it gets following instead of drag to place signature option.

**Review by thefoodiecoder — CHANGES_REQUESTED** [review]
Need changes in example to guide user better towards setting co-ordinates.

**Review by thefoodiecoder — APPROVED** [review]

**Pull request merged**